config/config-network.yaml

```diff
@@ -188,3 +188,35 @@ data:
     # NOTE: This flag is in an alpha state and is mostly here to enable internal testing
     #       for now. Use with caution.
     activator-san: ""
+
+    # The server certificates to serve the TLS traffic from ingress to activator.
+    # It is specified by the secret name, which has the "tls.crt" and "tls.key" data field.
+    # Use an empty value to disable the feature (default).
+    #
+    # NOTE: This flag is in an alpha state and is mostly here to enable internal testing
+    #       for now. Use with caution.
+    activator-server-certs: ""
```

Comment:
How about `activator-cert-secret` if we need this to be configurable? (I'm okay with it being configurable, though I'm not sure how useful that feature might be vs simply having a hard-coded name.)

Reply:
I wanted make this configurable as activator can turn on the https serving or not based on the config as:

```golang
	if networkConfig.ActivatorServerCert != "" {
              // activator starts serving https.
        }
```

Reply:
For the name of `activator-cert-secret` vs `activator-server-certs`, we already have the activator ca as `activator-ca` (without `-secert`) so it should be consistent?

Reply:
I'd thought the `activator-ca` was the actually CA certificate (which doesn't need to be kept private, because it's a certificate, not keys).

I'd prefer to have the `-secret` name here, because it provides a clearer clue that this is a value that needs to be kept private as well.